config.json

```diff
@@ -507,6 +508,14 @@
       "difficulty": 1,
       "topics": [
       ]
+    },
+    {
+      "slug": "proverb",
+      "difficulty": 1,
+      "topics": [
```

Comment:
In case you're not sure what topics include here, you can find a list of topics in https://github.com/exercism/x-common/blob/master/TOPICS.txt